drv/sprot-api/src/lib.rs

```diff
@@ -344,6 +345,15 @@ pub enum ReqBody {
     Attest(AttestReq),
     // Added in sprot protocol version 4
     RotPage { page: RotPage },
+    // Added in sprot protocol version 5
+    Swd(SwdReq),
+}
+
+// Added in sprot protocol version 5
+#[derive(Clone, Serialize, Deserialize, SerializedSize)]
+pub enum SwdReq {
+    EnableSpSlotWatchdog { time_ms: u32 },
+    DisableSpSlotWatchdog,
```

Comment:
Do we want a query or return previous state on an action?

Reply:
I thought about this, but it seems easier to make it idempotent.

MGS will retry if packets are lost, so idempotent functions are easier to think about.  For example if MGS is trying to disable the watchdog, misses a `DisableSpSlotWatchdogAck` and sends a second `DisableSpSlotWatchdog`, having the functions be idempotent prevents a spurious error.